core/modules/block/block.admin.inc

```diff
@@ -59,12 +64,14 @@ function block_admin_list() {
     $rows[] = $row;
   }
 
-  return array(
+  $block_admin_list = array(
     '#theme' => 'table__block_admin_list',
-    '#rows' => $rows,
     '#header' => $header,
+    '#rows' => $rows,
     '#empty' => t('No custom blocks have been created yet.'),
   );
+
+  return render($help_text) . render($block_admin_list);
```

Comment:
Since you are creating two render arrays that contain the key `#theme`, it's more efficient to return the arrays and let the Theme layer take care of rendering them, rather than invoke `render` here. It also allows contrib modules to modify the output before it's rendered. 

I suggest changing this to:

```php
  return array('help_text' => $help_text, 'block_admin_list' => $block_admin_list);
```

Reply:
Of course 👍🏼